src/sampling.rkt

```diff
@@ -132,9 +132,9 @@
      [err
       (values err precision +nan.0)]
      [(not err?)
-      (define infinite?
+     (define infinite?
       (ival-lo (is-infinite-interval repr (apply ival-or exs))))
-      (values (if infinite? 'infinite 'valid) precision exs)
+     (values (if infinite? 'infinite 'valid) precision exs)
```

Comment:
I think the original indentation is good.

Reply:
🤦‍♂️

Reply:
Bruh, you gotta set up proper indentation for Racket.